OpenRA.Mods.D2k/Traits/MakesNoise.cs

```diff
@@ -0,0 +1,36 @@
+#region Copyright & License Information
+/*
+* Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+* This file is part of OpenRA, which is free software. It is made
+* available to you under the terms of the GNU General Public License
+* as published by the Free Software Foundation. For more information,
+* see COPYING.
+*/
+#endregion
+
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.D2k.Traits
+{
+	[Desc("This actor makes noise.")]
+	public class MakesNoiseInfo : ITraitInfo
+	{
+		[Desc("How much noise does this actor produce.")]
+		public readonly int Intensity = 0;
+
+		[Desc("How much does the sound subside with each unit of distance (each cell).")]
```

Comment:
I see no reason to say both "each unit of distance" and clarify "each cell", instead of just saying "with each cell".